scripts/vnv-boot-collators/init-env.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Avoid committing the secrets
+export COLLATOR1_OVERALL_SECRET=""
```

Comment:
Are these `_SECRET` values items that might be useful to read from the environment?

```
export COLLATOR1_OVERALL_SECRET="${COLLATOR1_OVERALL_SECRET}"
```

The above should allow the value to be populated by a predefined value.